src/java/org/apache/cassandra/transport/SimpleClientSslContextFactory.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
yeah that's a good point.  I think I might have done this because it was adjacent to `SimpleClient`, but I agree that it should live with tests because I don't think we would want it to be used outside of that context.